lib/sorbet-rails/routes_rbi_formatter.rb

```diff
@@ -31,6 +31,10 @@ def header(routes)
     @parlour.root.create_module('ActionView::Helpers') do |mod|
       mod.create_include('GeneratedUrlHelpers')
     end
+
+    @parlour.root.create_class('ActionMailer::Base') do |mod|
```

Comment:
Now I think about it, it's probably not. As far as I'm aware sorbet doesn't run in views, since they don't have `.rb` extensions. Which is why it probably hasn't come up before.